app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,17 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag '//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/le-frog/jquery-ui.min.css' %>
+    <%= favicon_link_tag asset_path('favicon.png') %>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="container">
```

Comment:
Again minor stuff but here you can use a `main` element instead of `div`. Doesn't change the functionality or anything but it uses the html5 conventions as it indicates that there will be the main content.

Reply:
Thanks for reviewing!